patchwork/migrations/0048_patch_has_planned_review_and_more.py

```diff
@@ -0,0 +1,71 @@
+# Generated by Django 5.0.4 on 2024-04-16 04:31
+
+import datetime
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('patchwork', '0047_add_database_indexes'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
I'm guessing this was auto-generated? I didn't see it pop up in the most recent series. Could you regenerate this?